backend/app/services/adapters/task_kinds/queries.py

```
@@ -37,32 +45,166 @@
+        skip: Number of items to skip
+        limit: Maximum number of items to return
+
+    Returns:
+        Filtered and paginated list of TaskResource objects
+    """
+    # Filter tasks for display
+    filtered_tasks = []
+    for task in tasks:
+        try:
+            task_crd = Task.model_validate(task.json)
+        except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)